.autocorrectignore

```diff
@@ -5,4 +5,5 @@
 files/
 docs/
```

Comment:
I'm not sure could we ignore all path except the sub dir for zh-CN by this (as we are now only enable this for zh-CN), so we can simplify the script below

```suggestion
/*
```